# Global Health and Life Expectancy Analysis  
## Fourth Notebook: Visual Synthesis of Structural Health Disparities

Building on the analytical results established in Notebook 3, this notebook provides a **visual synthesis** of global health disparities across countries and regions. Rather than introducing additional statistical models or hypothesis tests, the objective here is to **translate previously established findings into clear, interpretable visual evidence**.

The focus is on **communicating structure rather than estimation**. Through a set of carefully designed visualizations, this notebook illustrates how differences in population health are organized spatially and temporally, and how these differences persist over time despite overall global improvements.

The year **2015** is used as a reference point to anchor cross-sectional comparisons, while the **2000–2015 period** is used to contextualize these patterns within longer-term regional trajectories. Together, these perspectives provide a coherent visual answer to the central question of whether observed health disparities are transient or structurally embedded.

### Specifically, this notebook aims to:
- Visually compare overall population health across regions and development levels in 2015.
- Illustrate the spatial organization of global health outcomes.
- Highlight the role of education as a key organizing dimension of population health.

This notebook is intended as a **communication layer** rather than an analytical extension. All figures are descriptive by design and should be interpreted as visual summaries of patterns formally established in prior notebooks.

**Author:** J-F Jutras  
**Date:** December 2025  

**Datasets:**  
- `final_dataset_for_notebook3.csv` — cross-sectional reference data (2015)  
- `df_original_PC1.csv` — panel data with composite health index (2000–2015)


## 4.1-Data Loading and Overview

In [1]:
import kagglehub
import os
import pandas as pd

#Download first dataset
df = pd.read_csv('/kaggle/input/final-dataset-who/final_dataset_for_notebook3.csv')
df.head(5)

,Country,Region,Year,Economy_status_Developed,Economy_status_Developing,PC1_health_index,Alcohol_consumption,Incidents_HIV,GDP_per_capita,Population_mln,Schooling
0,Turkiye,Middle East,2015,0,1,-1.280928,1.32,0.08,11006,78.53,7.8
1,Spain,European Union,2015,1,0,-1.726479,10.35,0.09,25742,46.44,9.7
2,Russian Federation,Rest of Europe,2015,0,1,0.299416,8.06,0.08,9313,144.10,12.0
3,Cameroon,Africa,2015,0,1,2.610301,4.55,1.12,1383,23.30,6.1
4,"Gambia, The",Africa,2015,0,1,1.891440,2.69,0.96,661,2.09,3.4


In [2]:
#Download second dataset
df = pd.read_csv('/kaggle/input/final-dataset-who-pc1/df_original_PC1.csv')
df.head(5)

,Country,Region,Year,Infant_deaths,Under_five_deaths,Adult_mortality,Alcohol_consumption,Hepatitis_B,Measles,BMI,...,Incidents_HIV,GDP_per_capita,Population_mln,Thinness_ten_nineteen_years,Thinness_five_nine_years,Schooling,Economy_status_Developed,Economy_status_Developing,Life_expectancy,PC1_health_index
0,Turkiye,Middle East,2015,11.1,13.0,105.8240,1.32,97,65,27.8,...,0.08,11006,78.53,4.9,4.8,7.8,0,1,76.5,-1.585892
1,Spain,European Union,2015,2.7,3.3,57.9025,10.35,97,94,26.0,...,0.09,25742,46.44,0.6,0.5,9.7,1,0,82.8,-1.850609
2,India,Asia,2007,51.5,67.9,201.0765,1.57,60,35,21.2,...,0.13,1076,1183.21,27.1,28.0,5.0,0,1,65.4,1.138898
3,Guyana,South America,2006,32.8,40.5,222.1965,5.68,93,74,25.3,...,0.79,4146,0.75,5.7,5.5,7.9,0,1,67.0,0.220866
4,Israel,Middle East,2012,3.4,4.3,57.9510,2.89,97,89,27.0,...,0.08,33995,7.91,1.2,1.1,12.8,1,0,81.7,-2.002792


## 4.2-Statistical Confirmation of Structural Health Disparities

The analyses conducted in Notebook 3 provide the **statistical foundation** for interpreting the descriptive results in Notebook 4. Notebook 3 examined population health differences using both **cross-sectional data for 2015** and **panel data from 2000 to 2015**. Key points supporting subsequent analyses are summarized below.

---

**Significant Differences by Development Status**
- Developed vs. developing countries show a **large, statistically significant separation** in the composite health index (PC1_health_index) in 2015 (**U = 479, p < 0.001; δ = −0.82**).  
- Developed countries consistently display **lower PC1 values**, corresponding to better overall population health.  
- Although robust regression adjusting for Schooling was applied, the high negative Spearman correlation between PC1_health_index and Schooling (**ρ ≈ −0.74**) limits interpretability. *The adjusted estimates are influenced by multicollinearity, so differences cannot be fully attributed to education alone.*  
- The model mainly demonstrates the **methodological approach under non-normality and outliers**, rather than a causal decomposition.

---

**Regional Disparities Are Statistically Robust**
- Kruskal–Wallis tests reveal highly significant differences in health outcomes across regions (**H = 108.09, p < 0.001**).  
- Post-hoc Dunn’s tests identify which regions differ significantly:  
  - **Africa and the Middle East** systematically have worse outcomes  
  - **Europe, North America, and Oceania** have relatively better and similar outcomes  
- These findings justify **visual comparisons across regions** and identification of outlier countries.

---

**Temporal and Panel Analyses Confirm Structural Patterns**
- From 2000 to 2015, regional trajectories of PC1_health_index remain relatively stable.  
- **Between-region variance (≈ 1.04)** dominates **within-region fluctuations over time (≈ 0.067)**.  
- Normalized trends show that regional rankings are persistent, indicating that **2015 differences reflect enduring structural disparities**.  
- Panel regressions with fixed effects further demonstrate **robust associations between education (Schooling) and regional health**, controlling for unobserved time-invariant characteristics.

---

**Methodological Rigor Supports Reliable Interpretation**
- **PCA** summarizes correlated health indicators (life expectancy, adult mortality, BMI) into a **single composite index**, mitigating multicollinearity.  
- Non-parametric tests, robust regression, and panel methods account for **non-normality, heteroskedasticity, and outliers**, ensuring that conclusions drawn from both **cross-sectional (2015)** and **long-term panel analyses** are statistically valid.


## 4.3-How Economic Development Influences Health Indicators in 2015

Economic development is strongly associated with overall population health. Developed countries consistently show **better outcomes** across key indicators: higher life expectancy, lower adult mortality, healthier BMI, and higher levels of schooling. This is reflected in their **lower PC1_health_index values**.

**Key statistical findings:**
- Mann–Whitney U = 479, p < 0.001 → **large separation** between developed and developing countries  
- Cliff’s Delta = −0.82 → confirms magnitude of difference  
- Spearman correlation with schooling: ρ ≈ −0.74 → education is linked to health but cannot fully explain differences

**Regional patterns:**
- Europe, North America, Oceania → systematically better health outcomes  
- Africa, Middle East → worse health outcomes

**Structural disparities persist over time:**
- Panel analyses (2000–2015) show that these differences are **stable**, reflecting long-standing inequalities in development and health infrastructure.

---

**Quick Comparison Table: Developed vs. Developing Countries**

| Development Status | Life Expectancy | Adult Mortality | BMI           | Schooling | PC1 Index |
|-------------------|----------------|----------------|---------------|-----------|-----------|
| Developed         | High           | Low            | Healthy       | High      | Low       |
| Developing        | Low            | High           | Less Healthy  | Low       | High      |

> In summary, a country’s level of economic development strongly predicts its population health profile: more developed countries enjoy better outcomes across all key indicators, while less developed countries systematically lag behind.


## 4.4-Global Health Overview in 2015

In [3]:
#Regional summary table for 2015 with styling
regional_table_styled = (
    df[df['Year'] == 2015]
    .groupby(['Country', 'Region'], as_index=False)
    .agg(PC1_avg=('PC1_health_index', 'mean'))  
    .groupby('Region')['PC1_avg']
    .agg(
        q1=lambda x: x.quantile(0.25),
        median='median',
        q3=lambda x: x.quantile(0.75),
        min='min',
        max='max'
    )
    .reset_index()
    .style
    .background_gradient(
        cmap='RdYlGn_r',      
        subset=['q1', 'median', 'q3', 'min', 'max']
    )
    .set_caption(
        "Regional Distribution of the Composite Health Index (PC1), 2015<br>"
        "Lower values indicate better population health"
    )
    .format(precision=3)
)

regional_table_styled

,Region,q1,median,q3,min,max
0,Africa,0.736,1.307,1.707,-1.394,2.909
1,Asia,-0.765,-0.244,0.310,-1.420,0.939
2,Central America and Caribbean,-1.311,-1.033,-0.872,-1.815,0.655
3,European Union,-1.801,-1.651,-1.314,-2.071,-0.811
4,Middle East,-1.956,-1.619,-1.173,-2.359,0.637
5,North America,-1.950,-1.949,-1.712,-1.951,-1.476
6,Oceania,-2.030,-0.983,-0.564,-2.271,0.663
7,Rest of Europe,-1.649,-1.191,-0.914,-1.974,-0.256
8,South America,-1.343,-1.006,-0.819,-1.903,-0.224


In [4]:
import plotly.express as px

def plot_health_choropleth(df, country_col='Country', health_col='PC1_health_index', dev_col='Economy_status_Developed'):
    #Copy and map development status
    df = df.copy()
    df['Development_Status'] = df[dev_col].map({0: 'Developing', 1: 'Developed'})

    #Create choropleth
    fig = px.choropleth(
        df,
        locations=country_col,
        locationmode='country names',
        color=health_col,
        hover_name=country_col,
        hover_data=['Development_Status', 'GDP_per_capita', 'Schooling'],
        color_continuous_scale='RdYlGn_r',  
        title='Global Health in 2015 by Country (PC1 Health Index)'
    )

    #Layout adjustments
    fig.update_layout(
        geo=dict(showframe=False, showcoastlines=True, projection_type='natural earth'),
        coloraxis_colorbar=dict(title='PC1 Health Index')
    )

    return fig

#Plot map
fig = plot_health_choropleth(df)
fig.show()

The choropleth map above illustrates the **composite health index (PC1)** across countries in 2015, providing a snapshot of overall population health at a global scale. 

Lower values of the PC1 index (shown in green) indicate **better health outcomes**, characterized by higher life expectancy, lower adult mortality, and more favorable nutritional status. Conversely, higher values (shown in red) represent **poorer health outcomes**. 

From the map, we can see clear spatial patterns: many **high-income regions in Europe, North America, and Oceania** display lower PC1 values, signaling better population health. In contrast, parts of **Africa and South Asia** exhibit higher PC1 values, reflecting persistent health challenges.

This visualization highlights that even in 2015, **structural disparities in health remain strongly linked to geography and economic development**, setting the stage for deeper analyses of the factors driving these differences.


## 4.5-Regional Health Disparities in 2015

In [5]:
def plot_regional_health_box_2015(df, health_col='PC1_health_index', region_col='Region'):
    #Average PC1 by country for 2015
    df_country_avg = (
        df[df['Year'] == 2015]
        .groupby(['Country', region_col], as_index=False)
        .agg(PC1_avg=(health_col, 'mean'))
    )

    #Order regions by median health index
    region_order = df_country_avg.groupby(region_col)['PC1_avg'].median().sort_values().index

    #Create boxplot
    fig = px.box(
        df_country_avg,
        x=region_col,
        y='PC1_avg',
        color=region_col,
        category_orders={region_col: region_order},
        title="Regional Disparities in Global Health (Country-Averaged PC1, 2015)",
        labels={'PC1_avg': "PC1 Health Index (avg by country)", region_col: "Region"},
        hover_data=['Country']
    )

    #Layout adjustments
    fig.update_layout(
        showlegend=False,
        xaxis_title="Region",
        yaxis_title="PC1 Health Index",
        boxmode='group'
    )

    return fig

#Usage
fig_box_country_avg_2015 = plot_regional_health_box_2015(df)
fig_box_country_avg_2015.show()

In [6]:
#Use the aggregated df_country_avg
df_country_avg_2015 = df[df['Year'] == 2015].groupby(['Country', 'Region'], as_index=False)\
                                          .agg(PC1_avg=('PC1_health_index', 'mean'))

#Identify outliers within each region
outlier_rows_2015 = []
for region, group in df_country_avg_2015.groupby('Region'):
    q1 = group['PC1_avg'].quantile(0.25)
    q3 = group['PC1_avg'].quantile(0.75)
    iqr = q3 - q1
    lower = q1 - 1.5 * iqr
    upper = q3 + 1.5 * iqr
    outliers = group[(group['PC1_avg'] < lower) | (group['PC1_avg'] > upper)]
    outlier_rows_2015.append(outliers)

df_outliers_2015 = pd.concat(outlier_rows_2015).sort_values(['Region', 'PC1_avg'])

#Create a color-coded table
styled_outliers_table_2015 = (
    df_outliers_2015.style
    .background_gradient(
        cmap='RdYlGn_r',   
        subset=['PC1_avg']
    )
    .set_caption(
        "Outlier Countries in Regional Health Index (2015)\n<br>(Lower PC1_avg indicates better population health)"
    )
    .format({'PC1_avg': "{:.3f}"})
)

styled_outliers_table_2015

,Country,Region,PC1_avg
49,"Egypt, Arab Rep.",Africa,-1.394
110,Morocco,Africa,-1.289
163,Tunisia,Africa,-1.285
94,Libya,Africa,-1.187
2,Algeria,Africa,-1.094
140,Seychelles,Africa,-0.928
136,Sao Tome and Principe,Central America and Caribbean,0.035
70,Haiti,Central America and Caribbean,0.655
176,"Yemen, Rep.",Middle East,0.637


The distribution of the composite health index (PC1_health_index) highlights pronounced regional disparities. Lower values indicate better overall population health, summarizing life expectancy, adult mortality, and BMI.

- **North America** and the **European Union** exhibit the healthiest populations, with medians of -1.66 and -1.37, respectively. Their interquartile ranges are relatively narrow, indicating more homogeneous health outcomes across countries. No outlier countries were identified in these regions. 
- The **Middle East** displays intermediate health outcomes, with a median of -1.30. **Yemen, Rep.** stands out as an outlier with substantially poorer health outcomes relative to the regional norm.  
- **Central America and the Caribbean** have a median of -0.76, with **Haiti** and **Sao Tome and Principe** identified as outliers—Haiti with worse health outcomes and Sao Tome and Principe with better-than-typical outcomes for the region.  
- **Africa** shows the poorest overall health outcomes, with a median of 1.95 and a wide interquartile range. Several countries deviate from the regional median with better health outcomes: **Libya, Tunisia, Egypt, Morocco, Algeria, Seychelles**.  
- Other regions like **Asia**, **South America**, and **Oceania** display broader distributions with no extreme outliers in 2015.

The table of regional outliers complements the boxplots by highlighting the specific countries that deviate from regional norms, emphasizing that structural inequalities persist even within regions.

## 4.6-Health Differences by Development Status in 2015

In [7]:
import plotly.express as px

#Scatter plot: PC1 Health Index vs Schooling, colored by Development Status
df_plot = df.copy()
df_plot['Development_Status'] = df_plot['Economy_status_Developed'].map({0: 'Developing', 1: 'Developed'})

fig_scatter = px.scatter(
    df_plot,
    x='Schooling',
    y='PC1_health_index',
    color='Development_Status',
    hover_name='Country',
    hover_data=['Schooling', 'PC1_health_index'],
    title='PC1 Health Index vs Schooling by Development Status (2015)',
    labels={'Schooling':'Average Years of Schooling', 'PC1_health_index':'PC1 Health Index'},
    color_discrete_map={'Developing':'orange','Developed':'blue'}
)

fig_scatter.show()


Scatter plots show the relationship between population health (PC1) and schooling across countries. Developed countries cluster at lower PC1 values and higher schooling, reflecting better and more homogeneous health outcomes. Developing countries display higher PC1 values and lower schooling, with greater variability, highlighting persistent structural disparities.


## 4.7-Summary and Limitations

**Summary**

In 2015, population health disparities are strongly structured by economic development and geography. Developed countries show better overall health outcomes—higher life expectancy, lower mortality, more favorable BMI, and higher schooling while less developed regions systematically lag behind. These patterns are persistent over time (2000–2015) and are robust to non-normality, outliers, and regional heterogeneity.

**Limitations**  
- **Causality**: Analyses are descriptive and correlational; causal inference is limited. High correlation between education and health (ρ ≈ −0.74) prevents clear separation of effects.  
- **Composite index**: PC1_health_index summarizes multiple indicators, potentially masking variation in individual components.  
- **Data quality**: Health and education statistics may be subject to reporting errors or inconsistencies, especially in developing countries.  
- **Temporal coverage**: Panel data only spans 2000–2015; longer-term trends or recent developments are not captured.  
- **Outliers**: Extreme values influence summaries; robust methods mitigate but do not fully remove their impact.  
- **Regional aggregation**: Grouping countries into regions may hide intra-regional disparities.  
- **Model assumptions**: Robust regressions and non-parametric tests reduce sensitivity to non-normality, but residual confounding and multicollinearity remain.  
- **Policy relevance**: Findings describe structural patterns but do not directly identify actionable interventions.  